src/sagas.js

```diff
@@ -1,6 +1,8 @@
 import { all } from 'redux-saga/effects';
+import { sagas as orderHistorySagas } from './order-history';
 
 export default function* rootSaga() {
   yield all([
+    orderHistorySagas(),
```

Comment:
I'm realizing now that we're down here that all the files are named in the plural; but that that feels ab it confusing as the imports should be singular. The files may contain multiple sagas, reducers, selectors, etc, but the thing that gets exported is a saga, reducer, or selector, singular.

Reply:
Agreed.